.github/pull_request_template.md

```diff
@@ -1,35 +1,25 @@
-**Are you a current Fivetran customer?** 
-<!--- Please tell us your name, title and company -->
-
-**What change(s) does this PR introduce?** 
-<!--- Describe what changes your PR introduces to the package -->
-
-**Does this PR introduce a breaking change?**
-<!--- Does this PR introduce changes that -->
-<!--- Mark yes or no (eg. [x] Yes). If yes, link the issue. -->
-- [ ] Yes (please provide breaking change details below.)
-- [ ] No  (please provide explanation how the change is non breaking below.)
-
-**Is this PR in response to a previously created Issue**
-<!--- If an Issue was created it is helpful to track the progress by linking it in the PR. -->
-<!--- Mark yes or no (eg. [x] Yes). If yes, link the issue. -->
-- [ ] Yes, Issue [link issue number here]
-- [ ] No 
-
-**How did you test the PR changes?** 
-<!--- Proof of testing is required in order for the PR to be approved. -->
-<!--- To check a box, remove the space and insert an x in the box (eg. [x] CircleCi). --> 
-- [ ] CircleCi <!--- CircleCi testing is only applicable to Fivetran employees. --> 
-- [ ] Other (please provide additional testing details below)
-
-**Select which warehouse(s) were used to test the PR**
-<!--- To check a warehouse remove the space and insert an x in the box (eg. [x] Bigquery). --> 
-- [ ] BigQuery
-- [ ] Redshift
-- [ ] Snowflake
-- [ ] Postgres
-- [ ] Databricks
-- [ ] Other (provide details below)
+## PR Overview
+
+Please provide your name and company
```

Comment:
```suggestion
**Please provide your name and company**
```

Can you bold these in the default template.

Reply:
should i bold all the questions?

Reply:
Yeah that would be great. It makes for reading their responses much clearer